css/_app-content-list.scss

```diff
@@ -0,0 +1,127 @@
+/* app content list & detail view */
+/* TO BE MOVED TO CORE apps.css ONCE STANDARDIZED ACROSS APPS */
+/* DO NOT MAKE ANY CHANGES SPECIFIC TO CONTACTS HERE! */
+
```

Comment:
@skjnldsv please take a look at this part.
Please verify that all of this code is part of the app-content-list, that should be moved to core and that nothing is missing.

Reply:
It's planned. I'm working on it!